lib/ansible/roles/user/tasks/main.yml

```diff
@@ -9,8 +9,12 @@
   when:           has_vagrant.stdout
   sudo:           yes
 
+- name:           Create deploy group
+  group:          name=deploy state=present system=no
+  sudo:           yes
+
 - name:           Create deploy user
-  user:           name=deploy append=yes groups=www-data shell=/bin/bash comment="Created by Evolution WordPress"
+  user:           name=deploy append=yes group=deploy groups=www-data shell=/bin/bash comment="Created by Evolution WordPress"
```

Comment:
For future reference: linux will *implicitly* create a user-named group when creating a user...unless said group already exists, in which case it silently fails.

So, this is just us being super explicit:

1. create a `deploy` group
2. create a `deploy` user
3. set user's primary group to `deploy`
4. set additional groups of `www-data`